src/styles/Block.scss

```diff
@@ -0,0 +1,122 @@
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  border-radius: 5px;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+
+  &__imagelement {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    padding-top: 32px;
+  }
+
+  &__description {
+    line-height: 18px;
+    display: flex;
+    flex-wrap: wrap;
+    width: 166px;
+    box-sizing: border-box;
```

Comment:
remove this line because you will set to the body

Reply:
+